blockToAddresses.go

```diff
@@ -294,11 +356,14 @@ func getLogAddresses(addresses map[string]bool, logs *BlockLogs, blockNum string
         idx := leftZero(strconv.FormatInt(idxInt, 10), 5)
 
         blockAndIdx := "\t" + blockNum + "\t" + idx
-        
-        for j := 0 ; j < len(logs.Result[i].Topics); j++ {
+
+        for j := 0; j < len(logs.Result[i].Topics); j++ {
             addr := string(logs.Result[i].Topics[j][2:])
-            if (isPotentialAddress(addr)) {
-                addresses["0x" + string(addr[24:]) + blockAndIdx] = true
+            if isPotentialAddress(addr) {
```

Comment:
The fact that isPotentialAddress and isGood don't take in the same input is unexpected and also confusing; we should standardize, probably.